tests/runtime/out_stackdriver.c

```diff
@@ -16,7 +16,6 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
```

Comment:
Is this blank line deleted intentionally?

Reply:
Nope. I blame VSCode again. Fixed.